mlflow/mleap.py

```diff
@@ -243,7 +244,10 @@ def add_to_model(mlflow_model, path, spark_model, sample_input):
     os.makedirs(mleap_path_full)
 
     dataset = spark_model.transform(sample_input)
-    model_path = "file:{mp}".format(mp=mleap_datapath_full)
+    if os.name != "nt":
+        model_path = path_to_local_file_uri(mleap_datapath_full)
+    else:
+        model_path = mleap_datapath_full
```

Comment:
Does the mleap Bundler require the prefix 'file:/'? https://github.com/combust/mleap/blob/a2dbcee6860802da2fda7f4b14e1dd35129214aa/bundle-ml/src/main/scala/ml/combust/bundle/BundleFile.scala#L22-L37

Reply:
I've tried `file:`, `file:/`, and `file:///`, none of which work. Will dig in further to see if there's some alternative structure for Windows.

Reply:
I've tried `file:`, `file:/`, and `file:///`, none of which work. Will dig in further to see if there's some alternative structure for Windows.